vendor/engines/secure_rooms/app/controllers/secure_rooms_api/events_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
         ),
       )
 
-      render nothing: true
+      render head :ok
```

Comment:
This line is causing test failures that we're double-rendering--I'm not quite following why. Nothing else seems to be calling render in this action. Any insight?

Reply:
I'm not sure why you're not seeing this elsewhere, but it's just `head :ok`. No `render`.